scipy/signal/filter_design.py

```diff
@@ -321,6 +321,11 @@ def freqz(b, a=1, worN=512, whole=False, plot=None, fs=2*pi):
         radians/sample (so w is from 0 to pi).
 
         .. versionadded:: 1.2.0
+    include_nyquist : bool, optional
+        If `whole` is False and `worN` is an integer, setting `include_nyquist` to True
+        will include the last frequency (Nyquist frequency).
```

Comment:
For the sake of being explicit, you may want to add something like "and is otherwise ignored."

Reply:
@e-q Thank you for your review. I added a docstring  based on you comment.

Reply:
I think @e-q was talking about the `include_nyquist` variable being ignored under the conditions, rather than the Nyquist freq being ignored, i.e., doing at the end "... frequency (Nyquist frequency) and is otherwise ignored.".

If you really want to say something about the `False` condition, it would be better to say that it's "omitted" rather than "ignored", but I don't think we need to since it's implied.

Reply:
Oh, OK. I understand. I fixed the docstring.